HW#1-Radimir_Sychevsky/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>

```

Comment:
The widget has unnecessary scroll and white space in IE:
![image](https://user-images.githubusercontent.com/71435688/94620998-361c6280-02b8-11eb-9995-014b98554291.png)
In other browsers there is empty background too.